AsyncDisplayKit/ASCollectionNode.mm

```diff
@@ -459,6 +462,16 @@ - (nullable UICollectionViewCell *)cellForItemAtIndexPath:(NSIndexPath *)indexPa
   return [self.dataController contextForSection:section];
 }
 
+- (ASCollectionData *)createNewData
+{
+  ASCollectionData *data = [[ASCollectionData alloc] initWithReusableContentFromCompletedData:self.dataController.previousData];
+  __weak __typeof(self) weakSelf = self;
+  data.postNodeBlock = ^(ASCellNode *node) {
+    [weakSelf.view didCreateNode:node];
```

Comment:
Probably add assert main thread in this block for good measure, and assert isNodeLoaded?